### PR TITLE
style(Tree): visual improvements

### PR DESCRIPTION
**Pull request by leko91**

### CODE REVIEW

**Review by GoranRibic — APPROVED** [review]

File: src/components/Tree/TreeItem.tsx [hunk 1]
```diff
@@ -191,39 +191,51 @@ export const TreeItem = ({
...
+                        paddingLeft: paddingLeftByLevel,
                     }}
-                    onDrop={handleDrop}
-                    accept={getAcceptTypes(accepts ?? treeId, 'within')}
                 >
                     <div
                         ref={itemRef}
-                        className={merge([
-                            'tw-flex tw-py-2 tw-px-2.5 tw-no-underline tw-leading-5 tw-h-10',
-                            FOCUS_VISIBLE_STYLE,
```
Comment (SamuelAlev) [inline src/components/Tree/TreeItem.tsx:222]: The focus style needs to be on the item with tabindex

File: src/components/Tree/TreeItem.tsx [hunk 2]
```diff
@@ -191,39 +191,51 @@ export const TreeItem = ({
 
     const hasChildren = Children.count(enhancedChildren) > 0;
 
+    const paddingLeftByLevel = treeItemState && `${treeItemState.level > 0 ? treeItemState.level : 0}rem`;
+
```
Comment (SamuelAlev) [inline src/components/Tree/TreeItem.tsx:195]:
```suggestion
    const paddingLeftByLevel = `${treeItemState?.level ?? 0}rem`;

```

Comment (leko91) [reply src/components/Tree/TreeItem.tsx:195]: I thought root was -1 😄

Comment (SamuelAlev) [reply src/components/Tree/TreeItem.tsx:195]: Root doesn't use this component

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**